subsys/caf/modules/buttons.c

```diff
@@ -26,8 +26,13 @@
 #include <logging/log.h>
 LOG_MODULE_REGISTER(MODULE, CONFIG_CAF_BUTTONS_LOG_LEVEL);
 
+#ifndef CONFIG_CAF_BUTTONS_PM_WAKEUP_TIMEOUT
+#define CONFIG_CAF_BUTTONS_PM_WAKEUP_TIMEOUT 0
```

Comment:
I really do not like if CONFIG_ setting are overwritten this way.
Use rather:
```
#ifdef CONFIG_CAF_BUTTONS_PM_WAKEUP_TIMEOUT
#define BUTTONS_PM_WAKEUP_TIMEOUT CONFIG_CAF_BUTTONS_PM_WAKEUP_TIMEOUT
#else
#define BUTTONS_PM_WAKEUP_TIMEOUT 0
#endif
```
And then use BUTTONS_PM_WAKEUP_TIMEOUT in the code.